content/en/agent/faq/linux-agent-2022-key-rotation.md

```diff
@@ -116,6 +116,8 @@ Otherwise, the command returns a non-0 exit code and the following output:
 package gpg-pubkey-fd4bf915 is not installed
 ```
 
+Alternatively, you can check if you `datadog.repo` file contains https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public as one of the `gpgkey` entries. This key file will be updated with the new key as soon as it is in use.
```

Comment:
```suggestion
Alternatively, check if your `datadog.repo` file contains `https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public` as one of the `gpgkey` entries. This key file is updated with the new key as soon as it is in use.
```

Reply:
I still think we should say "will be updated" instead of "is updated" since this will be a one-shot thing that will happen in June this year (and we are asking customers to be prepared for it).